### DISCUSSION TITLE
Glow for just specific materials?

### DISCUSSION THREAD

**Discussion by EnrichoUser** [bnpr/Malt#347] [Q&A] [answered]
Hello, is there a way to make only specific materials/nodes have a glow effect rather than all materials in the scene having it?

**Comment by pragma37** [comment, marked as answer]
You can create a new AOV for your intended bloom color and use that instead of the main color:
![imagen](https://user-images.githubusercontent.com/36610999/178055275-9e9722ef-c95b-449f-9af5-6942a8c30e6b.png)
[bloom-sample.zip](https://github.com/bnpr/Malt/files/9074572/bloom-sample.zip)

**Reply by EnrichoUser** [reply]
Thank you.

**Comment by Idefix20** [comment]
Newbie here. I can open this file with no problems in Blender 4.2. However, I can not recreate the nodes when trying to apply it to a new file.
When I try to append or copy the material nodes, the nodes are all blank ( there are no inputs) 

What is the difference between the Bloom Material Node tree and the Default Mesh?

How the I recreate the former as I can somewhat recreate the latter?

**Reply by pragma37** [reply]
Save the bloom node file with the current Blender/Malt version first. Then you can append it.

The Bloom Material is a Screen Material and the Default Mesh is a Mesh Material.
https://malt3d.com/Documentation/Graphs/#graph-types
